src/extension.ts

```diff
@@ -188,18 +217,18 @@ export function hideStatusLine(): void {
 }
 
 // start the integration file
-async function runTheFile(context: vscode.Uri) {
-	await startIntegration(context)
+async function runTheFile(...args: any[]) {
+	await startIntegration(args)
```

Comment:
better not mix await with then/catch

Reply:
Bad things happen in here trying to switch to try/catch.